### DISCUSSION TITLE
“You can't sign in to this app because Google can't confirm that it's safe”

### DISCUSSION THREAD

**Discussion by qnblackcat** [qnblackcat/uYouPlus#341] [General]
# ⚠️ UPDATE ⚠️: The issue has been addressed in [v17.37.3-2.1](https://github.com/qnblackcat/uYouPlus/releases/tag/v17.37.3-2.1) or higher. Please update to the latest version.

***
- Google has added a new implementation which prevents new users from signing in to modified YouTube app (see [Less secure apps](https://support.google.com/accounts/answer/6010255?hl=en)) - https://github.com/qnblackcat/uYouPlus/issues/319.
- Basically, If the bundle ID of the app is modified, Google won't let us sign in. If you're already signed it, you will be fine. Just don't remove the app so you don't have to sign in again. - ~~With that being said, some people are not affected by this issue at all~~. 
- Fortunately, there are two workarounds. See [FAQ](https://github.com/qnblackcat/uYouPlus/wiki/FAQ#q-you-cant-sign-in-to-this-app-because-google-cant-confirm-that-its-safe-warning-when-signing-in) for more details.

**Comment by DynamicVapor** [comment]
You can create a new YouTube account through the app and it will work as a sign-in currently, not exactly a work around but just a temporary Band-Aid until a fix is found.

Workaround found, currently working.

Step 1: Proceed to login on UYou app.
Step 2: When you get to error screen where it says unsafe app, press and hold on the “accounts.google” url at the top until you get copy option, copy the url. (Important to copy only url link when you get to error screen not before)
Step 3: Open safari and open an incognito tab, paste url and enter.
Step 4: Follow instructions on screen until open in YouTube prompt appears, open in YouTube.

if executed correctly, you will be signed in.

**Reply by drfdr** [reply]
nice,working. Managed to log in, no need to create a new account.

**Reply by zAlfredoo** [reply]
As of September 25th 2022 this no longer works.

**Comment by buitim** [comment]
I had to nuke the sideloaded app from my device recently and noticed that I was still logged in after a fresh install. I'm curious if this works for anyone, as I have not personally tried it myself: 

- Try to download the official YouTube app from the App Store and sign in there. 
- See if this account is still logged-in in another Google app on your phone. If it does, delete the YouTube app and sideload uYouPlus. 
- Open it and see if it prompts you to select your Google account. 

This was what basically worked for me... but I could just be one of the lucky unaffected individuals.

**Reply by DRECKINTERNET** [reply]
this had worked for me verry long but today i got logged out from uyou and dont know why

**Reply by Morku** [reply]
> I had to nuke the sideloaded app from my device recently and noticed that I was still logged in after a fresh install.

I can confirm that. I wanted to have 17.30.3-2.1 and I press install, but it didn't (Signulous). So I uninstalled 17.26.2-2.1 and installed 17.30.3-2.1. I was still logged in, but my settings were reset. Strange.

**Reply by DRECKINTERNET** [reply]
if u uninstall uyou and reinstall it your app settings resetting
i didnt update it or something i just got logged out this morning and cannot login anymore

Why is youtube and and sideloading on android so easy and on ios the biggest shit ever existed

**Reply by Cornerstar31** [reply]
I got it a few minutes later so it was solved. I tried editing the keys by combining them, with the keys from a original YouTube app(AppStore). Didn’t click through when it showed app unsafe. But copied the user name page from Google. When copying the unsafe address. It finally worked.

**Comment by DRECKINTERNET** [comment]
i opened the uyou app and just got logged out and now im stuck on the login page 
![402DD3EA-16FC-4DB7-8DBD-E0B008F81074](https://user-images.githubusercontent.com/110664410/183045330-4ff05467-5eda-46a6-b04a-a6706caa41c3.png)

**Comment by podupuis** [comment]
Changing the bundle ID still doesn't work for me unfortunately

**Reply by qnblackcat** [reply]
> Changing the bundle ID still doesn't work for me unfortunately

Hm. Are you sure uYouPlus uses the default bundle ID? Try to download YouTube from AppStore. If you can have both YouTube and uYouPlus, then UYouPlus is not using the default bundle ID

**Reply by podupuis** [reply]
Yeah, I can still have both even if I change the bundle id to com.google.ios.youtube . Maybe the bundle ID from the appstore in canada is different?

**Reply by DRECKINTERNET** [reply]
to me it is the same i can have youtube and uyou at the same time on my phone. in sideloadly it uses the automatic bundle id.

**Reply by qnblackcat** [reply]
> Yeah, I can still have both even if I change the bundle id to com.google.ios.youtube . Maybe the bundle ID from the appstore in canada is different?

That’s mean something is wrong with your signing service

Anyway, a new solution was found, you could try this:
In the sign in page, tap and hold the URL, open it in safari and sign in like normal. It will ask you Open in YouTube > Done!!

![](https://media.discordapp.net/attachments/858657553781227570/1005051580506194001/IMG_20220805_165601.jpg)

**Reply by podupuis** [reply]
Still having the same problem. It's driving me crazy lol

**Reply by DRECKINTERNET** [reply]
try to restart your phone

**Reply by podupuis** [reply]
> 

I'm going to try that. Finger crossed.

**Reply by qnblackcat** [reply]
@podupuis you can’t even sign in to YouTube on Safari!?

**Reply by podupuis** [reply]
I'm using iOS 16, could this be the problem?

**Reply by podupuis** [reply]
> try to restart your phone

doesn't work

**Reply by podupuis** [reply]
> @podupuis you can’t even sign in to YouTube on Safari!?

Nope.
![Uploading 85EDBF1E-DE25-419C-A5CC-94628F92DDD8.png…]()

**Reply by DRECKINTERNET** [reply]
no im on ios16 too and it worked fine until this moring.

the problem is i was so dumb and reinstalled the app. after a restart i landed  on the log in page

**Reply by podupuis** [reply]
> @podupuis you can’t even sign in to YouTube on Safari!?

https://share.icloud.com/photos/0fdH8wxlz6TdLtD2_VUl6YWAg

**Reply by podupuis** [reply]
Doesn't work in Google Chrome (ios) too. Same problem.

**Reply by qnblackcat** [reply]
> Doesn't work in Google Chrome (ios) too. Same problem.

well, it was unexpected... I'm running out of idea tbh

**Reply by DRECKINTERNET** [reply]
is there a way to edit the ipa that you set the login data in the ipa and install it with login information on the phone?

**Reply by DRECKINTERNET** [reply]
@podupuis in google there is an option that called "Acess less secure Apps" for me it is blocked. can you check if it works for you?
![image](https://user-images.githubusercontent.com/110664410/183063188-66cec8d0-cd80-4493-877b-8f9d56263d32.png)

**Reply by podupuis** [reply]
> @podupuis in google there is an option that called "Acess less secure Apps" for me it is blocked. can you check if it works for you?
> 
> ![image](https://user-images.githubusercontent.com/110664410/183063188-66cec8d0-cd80-4493-877b-8f9d56263d32.png)
> 
> 

Same problem as you. I think google no longer use this feature.

**Reply by qnblackcat** [reply]
@podupuis can you try again but this time you open the URL in a Safari incognito tab?

**Reply by DRECKINTERNET** [reply]
> @podupuis can you try again but this time you open the URL in a Safari incognito tab?

i think this is why youtube login no longer works this feauture existed since end of may. 

it exists no longer

**Reply by qnblackcat** [reply]
@DRECKINTERNET but have you tried what I said?🤔 (ofc it works for me)

**Reply by DRECKINTERNET** [reply]
> @DRECKINTERNET but have you tried what I said?🤔 (ofc it works for me)

do you mean open in inconigto tab? yes
![78BF333B-58A5-4488-96CC-7028FBC65471](https://user-images.githubusercontent.com/110664410/183079454-43265bd9-9715-47d2-ab54-fb91818bbc5f.png)

**Reply by DRECKINTERNET** [reply]
> > Yeah, I can still have both even if I change the bundle id to com.google.ios.youtube . Maybe the bundle ID from the appstore in canada is different?
> 
> That’s mean something is wrong with your signing service
> 
> Anyway, a new solution was found, you could try this: In the sign in page, tap and hold the URL, open it in safari and sign in like normal. It will ask you Open in YouTube > Done!!
> 
> ![](https://camo.githubusercontent.com/be528e9123cba5a5348ddbf2912b70397c8a1e742d54df0a7a8d14bdfe152bfe/68747470733a2f2f6d656469612e646973636f72646170702e6e65742f6174746163686d656e74732f3835383635373535333738313232373537302f313030353035313538303530363139343030312f494d475f32303232303830355f3136353630312e6a7067)

i tried this again and it worked this time but the youtube app redirects me back to my browser

**Reply by qnblackcat** [reply]
@DRECKINTERNET try to follow what I did [here](https://drive.google.com/file/d/17c9I419tdBf91oTeL0MQGNm4KlrVmlMk/view?usp=drivesdk)

**Reply by DRECKINTERNET** [reply]
> @DRECKINTERNET try to follow what I did [here](https://drive.google.com/file/d/17c9I419tdBf91oTeL0MQGNm4KlrVmlMk/view?usp=drivesdk)

thank you i dont know what i did wrong all the time because when i try this it usually just dont work or something but now it worked

**Reply by podupuis** [reply]
> @DRECKINTERNET try to follow what I did [here](https://drive.google.com/file/d/17c9I419tdBf91oTeL0MQGNm4KlrVmlMk/view?usp=drivesdk)

I can confirm! It's working now!

**Reply by podupuis** [reply]
So the working method is: try to login ---> only when you get the error, you then copy the link ---> open in safari ---> login to your YouTube account ---> click on open in YouTube ---> done ✅

**Reply by ghost** [reply]
the bundle id for altstore is com.google.ios.youtube.\<randomidmadebyaltstore\>

**Comment by qnblackcat** [comment]
Here is a new workaround, leave a feedback if it works for you:

In the sign in page, tap and hold the URL to copy it >  open it in Safari and sign in like normal > It will ask you Open in YouTube > Done!!

![](https://media.discordapp.net/attachments/858657553781227570/1005051580506194001/IMG_20220805_165601.jpg)

https://github.com/qnblackcat/uYouPlus/discussions/341#discussioncomment-3332300

**Reply by DRECKINTERNET** [reply]
if i login in safari i get the same error i get in uyou. this app is not safe to log in

**Reply by floresce** [reply]
This worked for me! I did this in private browsing from the Safari browser, so I can sign in to my Google account separately.

**Reply by mbucc9000** [reply]
Worked for me! THANKS

**Reply by themaster567** [reply]
This does not work for me, private browsing or not, even in a different browser.

**Reply by qnblackcat** [reply]
> This does not work for me, private browsing or not, even in a different browser.

Follow this video exactly: https://drive.google.com/file/d/17c9I419tdBf91oTeL0MQGNm4KlrVmlMk/view?usp=drivesdk

**Reply by darkbrow** [reply]
This worked for me. Thank you.

**Reply by royspace** [reply]
Work like a charm! Tysm

**Reply by karnadii** [reply]
work for me, I followed the video above, thanks

**Comment by thanhsang2707** [comment]
![6CF69AAC-5BB0-47F0-9FF4-10C44AF4DC23](https://user-images.githubusercontent.com/110694782/183149297-2621537a-144c-483b-9a9d-86e419540687.jpeg)
Copy URL and Open it in Private tab Safari > Sign in > It will ask Open Youtube > Done

**Reply by larsgrah** [reply]
That solution doesnt work anymore for me... even in safari i get the insecure warning

**Comment by Cornerstar31** [comment]
Doesn't work here either

**Reply by DRECKINTERNET** [reply]
do you have open in youtube app activated in safari? and do you use private tab?

**Comment by vincentallen2** [comment]
![CE1CA9AB-FB33-4806-A7C0-EE157FEACE70](https://user-images.githubusercontent.com/68659571/184280560-04890d94-1dae-4f1d-8b1e-e8a9933e9a4a.png)
![53D0D0B2-05C7-48F2-957F-6C8F33E97501](https://user-images.githubusercontent.com/68659571/184280567-c4d21c59-362a-46d1-8893-236b5ceab765.png)
Even with the copy url workaround, I’m still not able to sign in unfortunately. 
once I press open in YouTube, youtube open but asks which browser do I want to open the link.
At that point, no matter which browser I select, the browser states that the server could not be found

**Reply by qnblackcat** [reply]
… please follow exactly step by step: https://github.com/qnblackcat/uYouPlus/wiki/FAQ#q-you-cant-sign-in-to-this-app-because-google-cant-confirm-that-its-safe-warning-when-signing-in

**Reply by GiacoAV** [reply]
Can reproduce, I have the same issue and am not able to login...
I'm getting redirected to a com.google.sso.[supposedlyanID] and my app is not getting logged-in...

**Reply by vincentallen2** [reply]
> … please follow exactly step by step: https://github.com/qnblackcat/uYouPlus/wiki/FAQ#q-you-cant-sign-in-to-this-app-because-google-cant-confirm-that-its-safe-warning-when-signing-in

I’ve followed it exactly, but it’s still has the same outcome

**Comment by WoodpeckerBaby** [comment]
I think if you use a Google Workspace or G Suite account to sign in, it will not block you as of yet.

**Comment by julioverne** [comment]
#398 This fixes this issue

**Comment by enscen** [comment]
I've just creat a new account and it log me in. So simple.

**Comment by Corentin34725** [comment]
Hi guys ! I just change my iphone X for an iphone 11 and i got the same problem. I got it one time on my iphone X, but it worked when i sign in with safari in private mode ! But today it won't work with my iphone 11, this is why i tried a lot of thing and only one thing worked for me !! I just install the oldest realse of uYou : https://github.com/qnblackcat/uYouPlus/releases/download/v16.43.2-2.1/uYouPlus_16.42.3_2.1.ipa

After this, all work, i don't know why but you should try it !

**Comment by Macleykun** [comment]
If you seem to have the login problem, try downloading this version (this is cerecubeplus, but i assume it doesn't matter)
https://github.com/qnblackcat/CercubePlus/releases/download/v17.13.3-5.3.9/CercubePlus_17.13.3_5.3.9.ipa
After you installed this one (altstore/trollstore). You should be able to login normally.

Then install the latest version of the app of your choosing. It'll replace that app but keep the login session :).

**Reply by ghost** [reply]
i tried every version of the app, different browsers and accounts, different login methods - still don't work (ios 16, if that matters)

**Reply by sklein35** [reply]
Same here. I keep trying periodically to see if there’s some timeout associated with the block, but it never gets through. Super frustrating.

**Comment by zAlfredoo** [comment]
Unfortunately none of the options above worked for me, I had all security like 2fa etc disabled to see if it made a difference but it didn't. I get the same error when i load the link in the safari as a private browser.
